modules/create-policy-from-system-policies-view.adoc

```diff
@@ -0,0 +1,70 @@
+= Creating a security policy from the system policies view
+
+You can create new security policies from the system policies view.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *System policies*.
+. Click *+ New Policy* below the filter box on the top right side.
+. Turn off the *Enable Policy* toggle if you want to create a policy but enable it later.
+. Fill in the following details about your policy in the *Policy Details* section:
+** Enter a *Name* for the policy.
+** Select a *Severity* level for this policy, either `Critical`, `High`, `Medium`, or `Low`.
+** Choose *Lifecycle Stages* to which your policy is applicable, from *Build*, *Deploy*, or *Runtime*.
+You can select more than one stage.
+*** Build-time policies apply to image fields such as CVEs and Dockerfile instructions.
+*** Deploy-time policies can include all build-time policy criteria but they can also include data from your cluster configurations, such as running in privileged mode or mounting the Docker socket.
+*** Runtime policies can include all build-time and deploy-time policy criteria but they can also include data about process executions during runtime.
+** Enter details about the policy in the *Policy Description* box.
+** Enter an explanation about why the policy exists in the *Rationale* box.
+** Enter steps to resolve violations of this policy in the *Remediation* box.
+** Select policy *Categories* you want to apply to this policy.
+** Select *Notifications* channels to forward alert notifications when a violation occurs for this policy.
++
+[NOTE]
+====
+You must integrate {product-title} with your notification provider, for example, webhooks, Jira, PagerDuty, Splunk, or others before you can forward alerts.
+====
+//TODO: add link to integrations section.
+
+** Use *Restrict to Scope* to enable this policy only for a specific cluster, a namespace, or a label.
+You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
+** Use *Exclude by Scope* to exclude deployments, clusters, namespaces, and labels you specify, it means that the policy will not apply to the entities that you select.
+You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
+However, you cannot use regular expressions for selecting deployments.
+** For *Excluded Images (Build Lifecycle only)*, select all images that you do not want to trigger a violation for.
++
+[NOTE]
+====
+The *Excluded Images* setting only applies when you check images in a continuous integration system with the *Build* lifecycle stage.
+It will not have any effect if you use this policy to check running deployments in the *Deploy* lifecycle stage or runtime activities in the *Runtime* lifecycle stage.
+====
+. In the *Policy Criteria* section, that you you want to trigger the policy for.
```

Comment:
I think you might have removed some words here. "configure the attributes that you you want to trigger the policy for" ?